jsapp/scss/components/_kobo.collection-asset-list.scss

```diff
@@ -143,8 +143,6 @@
     position: relative;
     overflow: visible;
 
-    &--visible {}
```

Comment:
I would guess that someone left these empty blocks as a hint for future developers. You're the one working on it, though, so if you want to get rid of them, that's fine with me :)